packages/admin-panel/src/autocomplete/Autocomplete.js

```diff
@@ -1,198 +1,133 @@
-/**
- * Tupaia MediTrak
- * Copyright (c) 2018 Beyond Essential Systems Pty Ltd
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
  */
 
 import React from 'react';
 import PropTypes from 'prop-types';
-import throttle from 'lodash.throttle';
-import { connect } from 'react-redux';
-import MuiChip from '@material-ui/core/Chip';
 import { createFilterOptions } from '@material-ui/lab/Autocomplete';
 import styled from 'styled-components';
-import { Autocomplete as AutocompleteBase } from '@tupaia/ui-components';
-import { getAutocompleteState } from './selectors';
-import { changeSelection, changeSearchTerm, clearState } from './actions';
+import MuiChip from '@material-ui/core/Chip';
+import { Autocomplete as UIAutocomplete } from '@tupaia/ui-components';
 
 const Chip = styled(MuiChip)`
   &:first-child {
     margin-left: 6px;
   }
 `;
 
-const getPlaceholder = (placeholder, selection) => {
-  if (selection && selection.length) {
-    return null;
-  }
-
-  if (placeholder) {
-    return Array.isArray(placeholder) ? placeholder.join(', ') : placeholder;
-  }
-  return 'Start typing to search';
-};
-
-const filter = createFilterOptions();
-
-const AutocompleteComponent = React.memo(
-  ({
+export const Autocomplete = props => {
```

Comment:
This components will accept options from props. This brings some flexibilities to fetch options either from Redux dispatch or use query.